.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: required
```

Comment:
We really can't run the foxbox daemon as root... Instead of using avahi_client_set_host_name() did you try to add a cname record like in http://www.avahi.org/wiki/Examples/PythonPublishAlias ? This works for me without being root.
You don't even need to use dbus hopefully, just getting a group and adding the record should work.